app/views/users/show.html.erb

```diff
@@ -1 +1,24 @@
-<h1>Here is a specific user</h1>
\ No newline at end of file
+<%# <h1 class="user-banner">Here is a specific user</h1> %>
+
+<div class="user-detail">
+    <%= render partial: "users/shared/user", locals: {user: @user} %>
+  <p class="bio">
+    <h6>Bio:</h6>
+    <span>
+      <%= @user.bio %>
+    </span>
+  </p>
+  <ul class="user-detail__posts">
+    <% @user.posts.each do |post| %>
+      <li class="user-detail__posts_info">
```

Comment:
Please use the `last_three_post` method from your model to make sure that the User show page renders the user's 3 most recent posts as per project requirements.

`<% @user.last_three_post.each |post| %>`